node/pkg/governor/governor_monitoring.go

```diff
@@ -325,6 +325,10 @@ func (gov *ChainGovernor) IsVAAEnqueued(msgId *publicrpcv1.MessageID) (bool, err
 	gov.mutex.Lock()
 	defer gov.mutex.Unlock()
 
+	if msgId == nil {
+		return false, fmt.Errorf("no message ID specified")
+	}
+
```

Comment:
Should EmitterChain similarly be nil checked?

Reply:
Discussed offline, this will default to zero if unset, which is not a nil ptr deref issue.